src/style.css

```diff
@@ -0,0 +1,85 @@
+@font-face {
+  font-family: Avenir;
+  src: url("./ttc/Avenir-Book.ttf");
+}
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  color: #3d4e61;
+  font-weight: 300;
+  font-family: Avenir, sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+.form {
+  margin: 20px 8px;
+}
+
+.container {
+  position: relative;
+  display: block;
+}
+
+.search-img-big {
+  position: absolute;
+  top: 25px;
+  left: 26px;
+}
+
+.search-img-small {
+  position: absolute;
+  height: 11px;
+  top: 15px;
+  left: 13px;
+}
+
+.big-s-input,
+.small-s-input {
+  width: 100%;
+
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+
+  color: inherit;
+  font-family: inherit;
+  font-weight: inherit;
+}
+
+::placeholder {
```

Comment:
better to use with some class and do not change all placeholders. It is like adding styles to tag